packages/opentelemetry-propagator-jaeger/README.md

```diff
@@ -0,0 +1,38 @@
+# OpenTelemetry Propagator Jaeger
+[![Gitter chat][gitter-image]][gitter-url]
+[![NPM Published Version][npm-img]][npm-url]
+[![dependencies][dependencies-image]][dependencies-url]
+[![devDependencies][devDependencies-image]][devDependencies-url]
+[![Apache License][license-image]][license-image]
+
+OpenTelemetry propagation Jaeger provide HTTP header propagation for systems that using Jaeger HTTP header format.
+Example of usage:
+```javascript
+const { NodeTracerRegistry } = require('@opentelemetry/node');
+const { JaegerHttpTraceFormat } = require('@opentelemetry/propagator-jaeger');
+
+const registry = new NodeTracerRegistry({
+  httpTextFormat: new JaegerHttpTraceFormat()
+});
+```
+
+
+## Useful links
+- For more information on OpenTelemetry, visit: <https://opentelemetry.io/>
+- For more about OpenTelemetry JavaScript: <https://github.com/open-telemetry/opentelemetry-js>
+- For help or feedback on this project, join us on [gitter][gitter-url]
+
+## License
+
+Apache 2.0 - See [LICENSE][license-url] for more information.
+
+[gitter-image]: https://badges.gitter.im/open-telemetry/opentelemetry-js.svg
+[gitter-url]: https://gitter.im/open-telemetry/opentelemetry-node?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge
+[license-url]: https://github.com/open-telemetry/opentelemetry-js/blob/master/LICENSE
+[license-image]: https://img.shields.io/badge/license-Apache_2.0-green.svg?style=flat
+[dependencies-image]: https://david-dm.org/open-telemetry/opentelemetry-js/status.svg?path=packages/opentelemetry-core
```

Comment:
you have wrong url, it should be `opentelemetry-propagator-jaeger` instead of `opentelemetry-core`